proposals/NNNN-vehicle-type-filter.md

```diff
@@ -0,0 +1,435 @@
+
+For example, A proprietary app of an OEM creates Router service and the mobile device is connected to another OEMs' SDL enabled IVI system. The user would perceive that the proprietary app is doing something in the background, when connected to SDL enabled IVI system from another OEM.
+
+Please refer below screenshots.
+
+- Notification 1: Created by OEM App.
+- Notification 2: Created by SDL Android SDK for `SDL Router Service`.
+
+__Note the app name being shown in both notifications. In the below picture, the `SDL App` is using the “SDL Router Service” created by the `OEM App`.__
+
+<img src="../assets/proposals/NNNN-vehicle-type-filter/two_noti.png" alt="App name in notifications" width="500"/>
+
+
+The only way to remove notifications from OEM App when it connects to another vehicle is to force stop both the services so, that both notifications are removed. In this case, other apps connected through the `OEM Apps'` router service will also get disconnected. This is not a recommended approach. If the OEM app wants to unregisters itself from the SDL enabled IVI system, it will stop the service created by an app. The `SDL router service` from the OEM app will keep running.
+
+Please refer below screenshot. The `SDL Router Service` is hosted by the OEM app, the `Notification 2` with the app name will be shown to the user, giving the impression that the `OEM App` is still working. If the OEM App starts showing notifications when connected to another OEM's vehicle, this would confuse users. They would probably think that `OEM App` on their mobile device is working with another OEMs SDL enabled IVI system. This proposal will define ways to mitigate this issue.
+
+<img src="../assets/proposals/NNNN-vehicle-type-filter/one_noti.png" alt="App name in SDL router service notifications" width="500"/>
+
+This proposal tries to address the router service notification issue by defining the vehicle type filter. So, we could have an app that could specifically connect to certain vehicles. For example, an App that is designed only for Mustang vehicles.
+
+### Configuring lock screen with Vehicle name or brand logo 
+
+ This would help apps to show the vehicle brand logo depending on vehicle type its connected too. In the current implementation, the lock screen configuration is set along with the SDL lifecycle/manager configuration. Hence while configuring the lock screen, the app does not know to which vehicle its connecting too. The apps cannot change the configuration of the lock screen once the app receives the `Register app interface` response with vehicle details. Some of the apps expressed their interest to show the vehicle brand logo on the lock screen of the app depending on vehicle type information, which is not possible with the current implementation of SDL due to reasons described above. 
```

Comment:
```suggestion
 This would help apps to show the vehicle brand logo depending on the vehicle type to which it's connected. In the current implementation, the lock screen configuration is set along with the SDL lifecycle/manager configuration. Hence while configuring the lock screen, the app does not know to which vehicle it's connecting. The apps cannot change the configuration of the lock screen once the app receives the `RegisterAppInterface` response with vehicle details. Some app partners have expressed their interest in showing the vehicle brand logo on the lock screen of the app depending on vehicle type information, which is not possible with the current implementation of SDL due to reasons described above. 
```